src/nomad_material_processing/__init__.py

```diff
@@ -209,6 +505,32 @@ class SubstrateCrystalProperties(CrystalProperties):
         Section describing any miscut of the substrate with respect to the substrate
         orientation.
         """,
+        repeats=True,
+    )
+
+
+class ElectricProperties(ArchiveSection):
```

Comment:
I would call it `ElectricalProperties` or `ElectronicProperties`, the later being what we use in the NOMAD properties taxonomy.